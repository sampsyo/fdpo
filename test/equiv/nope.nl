in left: 32;
in right: 32;
out res: 32;

res = left;
---
res = right;
