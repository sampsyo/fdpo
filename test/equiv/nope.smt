(let ((.def_0 (= prog1_res prog2_res))) (let ((.def_1 (not .def_0))) (let ((.def_2 (= prog1_right prog2_right))) (let ((.def_3 (= prog1_left prog2_left))) (let ((.def_4 (and .def_3 .def_2))) (let ((.def_5 (= prog2_res prog2_right))) (let ((.def_6 (= prog1_res prog1_left))) (let ((.def_7 (and .def_6 .def_5 .def_4 .def_1))) .def_7))))))))
