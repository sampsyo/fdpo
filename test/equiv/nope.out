not equivalent
inputs:
  left = 1
  right = 0
differing outputs:
  res = 1 vs. 0
